templates/zerver/emails/followup_day2.html

```diff
@@ -13,7 +13,7 @@
     {{ _("In Zulip, <b>streams</b> determine who gets a message. <b>Topics</b> tell you what the message is about.")}} {{ _("Using topics, you can read Zulip one conversation at a time. You'll see each message in context, no matter how many different discussions are going on.") }}
 </p>
 
-<img src="/static/images/help/streams-and-topics.png" alt="{{ _('Streams and topics in the Zulip app') }}"/>
+<img src="{{ static('images/help/streams-and-topics.png') }}" alt="{{ _('Streams and topics in the Zulip app') }}"/>
```

Comment:
I don't think this is going to work -- we need the URL for this image to be stable/unhashed, since the email contains a fixed URL even if the Zulip server is updated and then reinstalled without containing old URLs.

Reply:
Thanks for providing this context. That is very helpful. For now, I can see just one solution here - adding a new variable like `{{ email_images_base_url }}` to handle this URL. It won't take much time and will work. But if you have any other ideas, I would appreciate them.